### ISSUE TITLE
Initial Help Setup

### ISSUE THREAD

**Issue by pharva** [Smurf-IV/Elucidate#56]
I'm brand new to github, snapraid, and Elucidate.  I set up snapraid.conf yesterday and successfully sync'd the array.  I attempted to configure Elucidate this morning and ran into this error: 

13:50:37.7411 [ 1] FATAL: The config file is not valid.[C:\Users\justi\Documents\snapraid-11.3-windows-x64\snapraid.conf]
The number of content files must be at least one greater than the number of parity files. There should be at least 2 content files.
 - Please Edit the Settings and ensure no errors when saving! 

This is my snapraid.conf file: 
 https://paste.ubuntu.com/p/yckbg4y6Jk/

I thought that I'd configured the snapraid.conf file to have three content files.  Can you provide some guidance?  Thank you.

**Comment by Smurf-IV** [comment]
I had a quick look at the file: Have you edited the file before you pasted it ?
Can you zip the Logs produced by Elucidate to see if there is anything I can see in them please ?
C:\ProgramData\Elucidate\Logs\....

**Comment by pharva** [comment]
Smurf-IV,
I'm not sure what happened with the file, but when I went to sync snapraid tonight, it gave me the same error as Elucidate did about the number of content files.  When I opened the snapraid.conf file, there was only one instance of "content F:\snapraid.cont."  I adjusted it again to list three different .cont files and the  sync worked.  When I opened Elucidate this morning (after a computer restart), the error was gone.  I was sure that I had edited that file (snapraid.conf) correctly for Elucidate's error message, but apparently I was wrong. 

Thank you for the reply.
